src/index.html

```diff
@@ -1,13 +1,64 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="html">
+  <head class="head">
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="body">
+    <header class="header">
+    </header>
+    <main class="card">
+
+      <div data-qa="card" class="card__container">
+
+        <div class="card__image">
+          <img src="images/imac.jpeg"
+            alt="img"
+            class="card__img"
+          >
+        </div>
+
+        <p class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <div class="product">
+          <span class="product__text">Product code: 195434</span>
+        </div>
+
+        <div class="rating">
+          <div class="rating__stars">
+            <span class="rating__star"></span>
+            <span class="rating__star"></span>
+            <span class="rating__star"></span>
+            <span class="rating__star"></span>
+            <span class="rating__star"></span>
+          </div>
```

Comment:
<img width="771" alt="Screenshot 2022-08-18 at 17 16 19" src="https://user-images.githubusercontent.com/84275496/185417485-e493f75e-1bfd-47c7-853c-0a9e6a3a6a75.png">

Reply:
Make BEM block from stars, other elements are just BEM elements

Reply:
Also styles for BEM blocks should be in separate files